### ISSUE TITLE
Cycling key

### ISSUE THREAD

**Issue by algernon** [algernon/Akela#61]
A key, that is in some way, similar to tap-dance, and possibly built on it too: press a key, then the cycle key, and the keyboard sends a backspace, and replaces the symbol with another. Continue until the dictionary is exhausted.

For example:

`o`, then `Cycle` => `o`, `Backspace`, `ó`. Pressing the `Cycle` key again would delete the `ó`, and type `ö`. Another `Cycle`, and we're on `ő`. Another, and we start again with `o`.

Additional dictionary libraries can be built on top of this, but that's a separate issue.

This requires tap-dance (#51), and needs to remember the last pressed key too, to be able to find it in the dictionary. That means that tap-dance has to be built with this in mind.

**Comment by algernon** [comment]
Unblocking, tap-dance is in. May induce changes to tap-dance, though.

**Comment by algernon** [comment]
Turns out that tap-dance is an overkill for this. Did it without.
